src/net/transport.cpp

```diff
@@ -33,8 +33,16 @@ void connect(std::string address, int port,
             return;
         }
         if (settings.find("ssl") != settings.end()) {
-            connect_ssl(r->connected_bev, SslContext::get_client_ssl(),
-                        [=](Error err, bufferevent *bev) {
+            Var<SslContext> ssl_context;
+            if (settings.find("ca_bundle_path") != settings.end()) {
+                logger->debug("ssl: using custom ca_bundle_path");
+                ssl_context = Var<SslContext>(new SslContext(settings.at("ca_bundle_path")));
+            } else {
+                logger->debug("ssl: using default context");
+                ssl_context = Var<SslContext>(new SslContext());
```

Comment:
We could probably add a `SslContext::global()` factory that returns `Var<SslContext>` where the return value here is a singleton `Var<SslContext>` instantiated on the first call.

Saying this because, IIRC, construction of a new `SslContext` is not a lightweight operation.

What do you think?